modules/virt-cancelling-vm-migration-web.adoc

```diff
@@ -5,15 +5,19 @@
 [id="virt-cancelling-vm-migration-web_{context}"]
 = Cancelling live migration of a virtual machine instance in the web console
 
-A live migration of the virtual machine instance can be cancelled using the 
-Options menu {kebab} found on each virtual machine in the 
-*Workloads* -> *Virtual Machines* screen, or from the *Actions* menu 
+A live migration of the virtual machine instance can be cancelled using the
+Options menu {kebab} found on each virtual machine in the
+*Virtualization* -> *Virtual Machines* tab, or from the *Actions* menu
 on the *Virtual Machine Details* screen.
 
 .Procedure
 
-. In the {VirtProductName} console, click *Workloads* -> *Virtual Machines*.
-. You can cancel the migration from this screen, which makes it easier to perform actions on multiple virtual machines in the one screen, or from the *Virtual Machine Details* screen where you can view comprehensive details of the selected virtual machine:
+. In the {VirtProductName} console, click *Virtualization* from the side menu.
+. Click the *Virtual Machine* tab to display your virtual machines.
+. Cancel migration of your virtual machines. Using the *Virtual Machine* tab,
+you can perform actions on multiple virtual machines, or from the
+*Virtual Machine Details* screen where you can view comprehensive details
+of the selected virtual machine:
 ** Click the Options menu {kebab} at the end of virtual machine and select
 *Cancel Virtual Machine Migration*.
 ** Click the virtual machine name to open the *Virtual Machine Details*
```

Comment:
The first tab of the virtual machine is `Overview` on OCP 4.5, `Details` is the 2nd tab. Click the virtual machine name is opening the `Virtual Machine Overview`. The action menu is available on all tabs.